vendor/bat-native-ads/src/bat/ads/internal/processors/contextual/text_embedding/text_embedding_html_event_info.cc

```diff
@@ -0,0 +1,17 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/processors/contextual/text_embedding/text_embedding_html_event_info.h"
+
+namespace ads {
+
+TextEmbeddingHtmlEventInfo::TextEmbeddingHtmlEventInfo() = default;
+TextEmbeddingHtmlEventInfo::TextEmbeddingHtmlEventInfo(
+    const TextEmbeddingHtmlEventInfo& info) = default;
+TextEmbeddingHtmlEventInfo& TextEmbeddingHtmlEventInfo::operator=(
+    const TextEmbeddingHtmlEventInfo& info) = default;
+TextEmbeddingHtmlEventInfo::~TextEmbeddingHtmlEventInfo() = default;
```

Comment:
See comment for header

Reply:
Was able to remove `operator=` line here. Other lines were needed to build.